### PR TITLE
Feature/rebase master to init project

### PR DESCRIPTION
**Pull request by emmanuelquinton**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,5 +4,7 @@ tests:
 	pytest
 
 package:
-	python setup.py sdist
+	python setup.py sdist bdist_wheel
 
+push_tags:
+	git push --follow-tags
```
Comment (dcremonini) [inline Makefile]: You removed bdist_wheel

Comment (dcremonini) [inline Makefile]: This is the second conversation.

**Review by dcremonini — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (emmanuelquinton) [reply Makefile]: yes

Comment (emmanuelquinton) [reply Makefile]: ok

**Review by dcremonini — APPROVED** [review]

**Pull request merged**